src/tests/Common/Directory.Build.targets

```diff
@@ -146,7 +146,7 @@
           TargetDir="runtime-test/"/>
 
         <RunTimeDependencyCopyLocal
-          Include="$(MonoProjectRoot)\wasm\build\WasmApp.targets"
+          Include="$(MonoProjectRoot)\wasm\build\*"
```

Comment:
Would this include the `README.md` as well?

Reply:
Yeah, but that should be fine for now. Better than using specific wildcards here, and then updating when new files get added.